### ISSUE TITLE
Dancer2::Manual::Deployment does not mention using the behind_proxy configuration option

### ISSUE THREAD

**Issue by jasonblewis** [PerlDancer/Dancer2#1166]
Setting the behind_proxy setting should be mentioned in the doco as if you are using apache to do the proxying and ssl termination, d2 will be generating  http  links instead of https

**Comment by xsawyerx** [comment]
@jasonblewis Any chance for a PR on this? I would merge it right away.

**Comment by nunorc** [comment]
Tentative PR available: https://github.com/PerlDancer/Dancer2/pull/1189

**Comment by ambs** [comment]
Should be enough. Reopen if not. Thanks.

**Comment by whosgonna** [comment]
Hi,  it looks like this still isn't in the current [Dancer2::Manual::Deployment](https://metacpan.org/pod/release/CROMEDOME/Dancer2-0.206000/lib/Dancer2/Manual/Deployment.pod) documentation for Dancer2 (0.206000).  Is it possible to re-merge this?

**Comment by cromedome** [comment]
Done, thanks for bringing it to our attention. No idea what happened there.

**Comment by racke** [comment]
Yeah it is weird. Checked for the commit hash and also for the text change in master ... not there.

**Comment by whosgonna** [comment]
Slightly related, the config manual documentation for [behind_proxy](https://metacpan.org/pod/distribution/Dancer2/lib/Dancer2/Config.pod#behind_proxy-(boolean)) has a section for setting Apache config to use `X-Forwarded-Protocol` and `X-Forwarded-host`  that don't look to be in the deployment manual in this section.  I'm not using Apache mod_proxy anywhere, so I don't feel like I'd be an authoritative voice for adding the instructions, bit probably is relevant.  Related note, I am using Nginx as a reverse proxy, and the example on the deployment page does include this information.

**Comment by cromedome** [comment]
@racke @whosgonna: I merged this again a little while ago. 

@whosgonna: Thanks, I will leave this issue open until we resolve that. I like the suggestion.

**Comment by cromedome** [comment]
This is addressed in the new documentation coming in Dancer2 2.0.0. Resolving.
